plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -247,7 +247,7 @@ bool GSDeviceOGL::Create(GSWnd* wnd)
 	m_convert.cb = new GSUniformBufferOGL(g_convert_index, sizeof(ConvertConstantBuffer));
 	// Upload once and forget about it
 	ConvertConstantBuffer cb;
-	cb.ScalingFactor = GSVector4i(theApp.GetConfig("nativeres", 0) ? 1 : theApp.GetConfig("upscale_multiplier", 2));
+	cb.ScalingFactor = GSVector4i(theApp.GetConfig("upscale_multiplier", 1)==1 ? 1 : theApp.GetConfig("upscale_multiplier", 2));
```

Comment:
1 is for native resolution.

 just for clarification I'll explain the function, formerly we check whether native res is enabled. if it is then, we set the scaling factor to 1 for normal upscale math. if it isn't then, we set the value at upscale_multiplier for the scaling factor. (upscaled native resolutiions)